### PR TITLE
Manage packages with hpack.

### PR DESCRIPTION
**Pull request by andreyk0**
https://github.com/sol/hpack#readme

### CODE REVIEW

File: package.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+license: BSD3
+homepage: https://github.com/saksdirect/okta-aws-login
+
+ghc-options:
+  - -Wall
+  - -threaded
+
+dependencies:
+  - aeson
+  - aeson-casing
```
Comment (amir) [inline package.yaml]: can we pin a version here?

Comment (andreyk0) [reply package.yaml]: We could but I don't like version pinning, interferes with regular updates.
Stackage's versioned "slice" of packages is a nice reference point where everything compiles across the board.

Comment (amir) [reply package.yaml]: Yeah, I agree on the value that Stackage brings but I was wondering if we could pin the older version of `aeson-casing` to fix our current problem?

But if that's not the direction of this PR, what are the benefits of using hpack directly when stack has hpack integrated internally?

Comment (andreyk0) [reply package.yaml]: This is just an `hpack` change, I'm working on a separate PR to fix the casing problem.

Comment (andreyk0) [reply package.yaml]: Oh, and this *is* the way to make use of the stack's internal hpack integration. It uses package.yaml to auto-generate cabal file.

Comment (amir) [reply package.yaml]: You're right, I'm sorry. I don't know what I was thinking! I was even involved in a change of behaviour in Stack regarding its hpack usage: https://github.com/commercialhaskell/stack/pull/3856

Comment (andreyk0) [reply package.yaml]: Haha, no worries! So, no objections then?

Comment (amir) [reply package.yaml]: Non at all.

**Pull request merged**